lab-michelle/src/component/category-item/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import CategoryForm from '../category-form';
+// import uuid from 'uuid/v4';
+import {categoryUpdate, categoryDestroy} from '../../action/category-actions';
+
+class CategoryItem extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className = "category-item">
+      <h3>{this.props.category.title}</h3>
+      <button onClick={() => this.props.categoryDestroy(this.props.category)}>X</button>
+      <CategoryForm
+        buttonText="update"
+        onComplete = {this.props.categoryUpdate}
+        category = {this.props.category}/>
+      </div>
+    );
+  }
+}
+
+let mapStateToProps = state => {
+  return {
+    categories: state,
+  };
+};
+
+let mapDispatchToProps = (dispatch, getState) => {
+  return {
+    categoryUpdate: category => dispatch(categoryUpdate(category)),
+    categoryDestroy: category => dispatch(categoryDestroy(category)),
+  };
+};
+
+export default connect(null, mapDispatchToProps)(CategoryItem);
+
+// CategoryItem Component
+//
+// should display the category's name and budget
```

Comment:
delete these comments before submitting, or keep them in the README.md

also - you should be displaying a name and budget or price, right now you're only displaying a name